invenio_rdm_records/services/config.py

```diff
@@ -250,8 +251,8 @@ class RDMRecordServiceConfig(RecordServiceConfig, ConfiguratorMixin):
     """RDM record draft service config."""
 
     # Record and draft classes
-    record_cls = RDMRecord
-    draft_cls = RDMDraft
+    record_cls = FromConfig("RDM_RECORD_CLS", default=RDMRecord)
+    draft_cls = FromConfig("RDM_DRAFT_CLS", default=RDMDraft)
```

Comment:
The usual argument about making "individual attributes" vs. "entire classes" config-based applies to this PR.